contrib/test_jenkins.sh

```diff
@@ -974,6 +974,22 @@ test_ucp_dlopen() {
 	fi
 }
 
+test_ucm_hooks() {
+    echo "==== Running UCM Bistro hook test ===="
+    for i in $(seq 1 30); do
+        echo -n "."
```

Comment:
maybe print every iteration index in new line? not sure it will work well with log output redirection in CI...

Reply:
fixed